lib/linkedin-scraper/profile.rb

```diff
@@ -155,6 +155,24 @@ def recommended_visitors
       end
     end
 
+    def projects
+      @projects ||= @page.search('.background-projects/div').map do |project|
+        project = project.at('div')
+
+        p = {}
+        start_date, end_date = project.at('.projects-date').text.gsub(/\s+|\n/, ' ').strip.split(' – ') rescue nil
```

Comment:
Line is too long. [114/80]<br>Avoid using `rescue` in its modifier form.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.